modules/aws-outposts-environment-info-aws.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-environment-info-aws_{context}"]
+= Obtaining information from your AWS account
+
+You can use the AWS CLI (`aws`) to obtain information from your AWS account.
+
+[TIP]
+====
+You might find it convenient to store some or all of these values as environment variables by using the `export` command.
+====
+
+.Prerequisites
+
+* You have an AWS Outposts site with the required hardware setup complete.
+
+* Your AWS Outpost is connected to your AWS account.
+//Not sure how to talk about this, check AWS docs.
+* You have access to your AWS account by using the AWS CLI (`aws`) as an administrator.
+
+.Procedure
+
+. List the Outposts that are connected to your AWS account by running the following command:
++
+[source,terminal]
+----
+$ aws outposts list-outposts
+----
+
+. Retain the following values from the output of the `aws outposts list-outposts` command:
+
+** The Outpost ID.
+
+** The Amazon Resource Name (ARN) for the Outpost.
+
+** The AWS Outpost availability zone.
++
+[NOTE]
+====
+The output of the `aws outposts list-outposts` command includes two values related to the availability zone: `AvailabilityZone` and `AvailabilityZoneId`. The value you use to configure a compute machine set that creates compute machines in AWS Outposts is `AvailabilityZone`.
+====
+
+. Using the value of the Outpost ID, show the instance types that are available in your AWS Outpost by running the following command. Retain the values of the available instance types.
++
+[source,terminal]
+----
+$ aws outposts get-outpost-instance-types \
+  --outpost-id <outpost_id_value>
+----
+// I have guessed at this based on https://docs.aws.amazon.com/cli/latest/reference/outposts/list-outposts.html
+
+. Using the value of the ARN for the Outpost, show and retain the subnet ID for the Outpost by running the following command:
++
+[source,terminal]
+----
+$ aws describe-subnets --filters outpost-arn=<outpost_arn_value> --subnet-ids
```

Comment:
It seems we have an issue here, afaik:
- incorrect command. `describe-subnets` should be in `ec2` namespace
- Filter syntax seems to be incorrect
- iiuc ` --subnet-ids` requires the list of subnets
```suggestion
$ aws ec2 describe-subnets --filters Name=outpost-arn,Values<outpost_arn_value>
```

https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-subnets.html

Reply:
Thanks!